toc/working-groups/app-runtime-platform/working-agreements/2021-10-29-reviewer-workflows.md

```diff
@@ -0,0 +1,81 @@
+# Reviewer Workflows for Issues and PRs
+
+**Date of Forum to be Discussed:** Wednesday Nov 3, 2021 9 Pacific / 12 Eastern / 17 Britain / 18 Germany
+See the community calendar for more details.
+
+**Author:** Amelia Downs - @ameowlia
+
+## Problem
+Our working group has 70+ repos. In order to run a healthy open source community we need to make sure that we triage issues and merge PRs in a timely fashion.
+
+**As a PR author I want:**
+* To have my PR reviewed and merged in a timely manner.
+
+**As an issue author I want:**
+* To have guidance on how to fix my bug in a timely manner.
+
+**As an Approver I want:**
+* To know which PRs and issues I have to review.
+* I want to know what is expected of me as a reviewer.
+
+**As a Tech Lead I want:**
+* To know how many open issues and PRs there are.
+* To be able to assign reviews for open issues and PRs.
+
+## Solution
+#### PR + Issue Tracking Github Projects for Each Sub-Group
+Each sub-group of our working group has a github project.
+These projects will contain a card for each issue and PR for the subgroup.
+The tech lead is responsible for assigning reviewers.
+The reviewers are responsible for triaging the card until either they make it to the “Done” or “Needs Fix” swimlanes.
+The reviewer is not responsible for fixing the bug for a given issue.
+
+**Links to Github Projects**
+* [App Platform - Diego](https://github.com/orgs/cloudfoundry/projects/20)
+* [App Platform - Garden Containers](https://github.com/orgs/cloudfoundry/projects/23)
+* [App Platform - Logging and Metrics](https://github.com/orgs/cloudfoundry/projects/19)
+* [App Platform - Networking](https://github.com/orgs/cloudfoundry/projects/24)
+
+### Issue Workflow
+Below is the outline of a typical issue workflow. [See drawing](https://docs.google.com/drawings/d/1_W-Xk8pCUCv1la-rEuJAA8rE_ivMKrmxrmWyR7b0Tk4/edit?usp=sharing).
+<img src="https://i.ibb.co/BN2qS60/Screen-Shot-2021-10-28-at-5-37-39-PM.png">
+
+### PR Workflow
+Below is the outline of a typical PR workflow. [See drawing](https://docs.google.com/drawings/d/1BlIXESgk_Ycp9jRQdnY_v_qrgn2fGwCSJ-eiAssryfg/edit?usp=sharing).
+<img src="https://i.ibb.co/7vCsqBX/Screen-Shot-2021-10-28-at-5-38-59-PM.png">
+
+
+## FAQ
+
+❓ **How can I see what is assigned to me?**
+
+Go to the github project.
+Either click on your face on a card that is already assigned to you or
+add the following query param to the URL: `?card_filter_query=assignee:YOUR-GITHUB-USERNAME`
+
```

Comment:
Can we update this section to also include a link to https://github.com/issues/assigned?q=is%3Aopen+archived%3Afalse+org%3Acloudfoundry to see all the issues/PRs assigned to a person in a single view?

Reply:
✅